### ISSUE TITLE
Crash when entering Nether

### ISSUE THREAD

**Issue by Fortist** [TheRandomLabs/Vertical-End-Portals#4]
I took a random portal, placed it down and walked in. I loaded briefly into the Nether (Saw red fog) and then immediately crashed, citing this mod and random patches.

Both are the latest mods.
1.12.0.1
1.5.0.1

```
---- Minecraft Crash Report ----
// Who set us up the TNT?

Time: 2019-01-14 15:09:00 EST
Description: Exception in server tick loop

java.lang.IllegalArgumentException: Frame cannot be smaller than 3 blocks
    at com.therandomlabs.randomportals.api.frame.Frame.<init>(Frame.java:60)
    at com.therandomlabs.randomportals.api.frame.Frame.<init>(Frame.java:47)
    at com.therandomlabs.randomportals.world.RPOTeleporter.makePortal(RPOTeleporter.java:508)
    at com.therandomlabs.randomportals.world.RPOTeleporter.placeInPortal(RPOTeleporter.java:62)
    at com.therandomlabs.randomportals.world.RPOTeleporter.placeEntity(RPOTeleporter.java:563)
    at com.therandomlabs.randompatches.common.RPTeleporter.placeEntity(RPTeleporter.java:75)
    at net.minecraft.server.management.PlayerList.transferEntityToWorld(PlayerList.java:721)
    at net.minecraft.server.management.PlayerList.transferPlayerToDimension(PlayerList.java:624)
    at net.minecraft.entity.player.EntityPlayerMP.changeDimension(EntityPlayerMP.java:695)
    at net.minecraft.entity.Entity.changeDimension(Entity.java:2543)
    at com.therandomlabs.randomportals.handler.NetherPortalTeleportHandler.handle(NetherPortalTeleportHandler.java:172)
    at com.therandomlabs.randomportals.handler.NetherPortalTeleportHandler.onServerTick(NetherPortalTeleportHandler.java:126)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_2416_NetherPortalTeleportHandler_onServerTick_ServerTickEvent.invoke(.dynamic)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
    at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
    at net.minecraftforge.fml.common.FMLCommonHandler.onPostServerTick(FMLCommonHandler.java:266)
    at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:712)
    at net.minecraft.server.integrated.IntegratedServer.tick(IntegratedServer.java:185)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
    at java.lang.Thread.run(Unknown Source)
```

**Comment by TheRandomLabs** [comment]
I'm not sure how it's possible for the frame size to be below 3 blocks. Could I have a look at your portal type JSON?

**Comment by TheRandomLabs** [comment]
This should be fixed in 1.12.2-1.5.0.2.
